### ISSUE TITLE
Clarify license of package

### ISSUE THREAD

**Issue by non-Jedi** [dzop/emacs-jupyter#2]
Headers of both this package and emacs-zmq state that they are released under GPLv2+. Could you confirm this? If so, the FSF recommends including the license in the repo with the source code I believe. Thanks.

**Comment by dzop** [comment]
Yes they are both under GPLv2+. Thanks for the info, I will get around to doing that.
